src/Engine/Party.h

```diff
@@ -239,6 +239,14 @@ struct Party {
         return character - pCharacters.data();
     }
 
+    /*
+    * Return the Character pointer based on the index
+    * 
+    * @param characterIndex     index of the character
+    * @return                   A valid character or nullptr in case the index is out of bounds.
+    */
```

Comment:
Codestyle, doxy comments start with`/**` and go like this:
```
/**
 * Bla.
 *
 * Blabla.
 */
```
Note that there is a space before each `*`.